pkg/plugin/processor/builtin/impl/webhook/http.go

```diff
@@ -156,6 +163,23 @@ func (p *httpProcessor) Configure(ctx context.Context, m map[string]string) erro
 	return nil
 }
 
+func (p *httpProcessor) validateHeaders() error {
+	if p.config.Headers != nil {
+		_, ok := p.config.Headers["Content-Type"]
```

Comment:
Since this is user input, it might be a different case (lower, mixed, etc.)

Reply:
We should add tests to make sure we recognize different cases.